common-libs/common/src/Affinity.ts

```diff
@@ -238,6 +273,13 @@ export class Affinity {
         return { result: false, error }
       }
 
+      // send VC_VERIFIED* metrics when verification is successful
+      // TODO: also record failed verification?
+      this._sendVCVerifiedMetric(credential, parse(result.data.holder.id).did)
+
+      // TODO: how to get the verifier did?
+      this._sendVCVerifiedPerPartyMetric(credential, verifierDid)
```

Comment:
@everdark there is no way how to pull verifier did here - this event could not be tracked on this level and still left only on SDK side

Reply:
👍